scpred_train_test_split.R

```diff
@@ -84,7 +90,8 @@ set.seed(opt$random_seed)
 print(opt$input_sce_object)
 # preprocess data from SCE object 
 sce_object = readRDS(opt$input_sce_object)
-sce_counts = as.matrix(normcounts(sce_object))
+sce_counts = as.matrix(assays(sce_object)[[opt$normalised_counts_slot]])
```

Comment:
Can you add a check that the slot exists, and exit with an error if it doesn't?

Reply:
@pinin4fjords done